lmdeploy/utils.py

```diff
@@ -55,13 +56,16 @@ def get_logger(
         file_handler = logging.FileHandler(log_file, file_mode)
         handlers.append(file_handler)
 
+    from mmengine.logging.logger import FilterDuplicateWarning
```

Comment:
could we copy `FilterDuplicateWarning` from mmengine to here and do not depend on mmengine?